src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -296,7 +296,7 @@ protected function instantiateObject(array &$data, $class, array &$context, \Ref
             if (__CLASS__ !== get_class($this)) {
                 $r = new \ReflectionMethod($this, __FUNCTION__);
                 if (__CLASS__ !== $r->getDeclaringClass()->getName()) {
-                    @trigger_error(sprintf('Method %s() will have a 6th `$format = null` argument in version 4.0. Not defining it is deprecated since 3.2.', get_class($this), __FUNCTION__), E_USER_DEPRECATED);
+                    @trigger_error(sprintf('Method %s::%s() will have a 6th `$format = null` argument in version 4.0. Not defining it is deprecated since 3.2.', get_class($this), __FUNCTION__), E_USER_DEPRECATED);
```

Comment:
@HeahDude organic analyses failure: `get_class($this) !== __CLASS__` when $this is a child class of AbstractNormalizer (which it always is in this case ;) )